hnn_core/network.py

```diff
@@ -1253,23 +1253,77 @@ def add_connection(self, src_gids, target_gids, loc, receptor,
 
         self.connectivity.append(deepcopy(conn))
 
-    def clear_connectivity(self):
-        """Remove all connections defined in Network.connectivity
+    def _get_expected_connectivities(self, src_types='all'):
```

Comment:
Since this is solely for testing purposes, should this function be moved to `test_network.py`?

Reply:
yes, let's move this to `test_network.py`